CMakeLists.txt

```diff
@@ -97,6 +91,13 @@ IF(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
     ELSE()
         MESSAGE(WARNING "\nCannot determine the version of the compiler selected to build GNU Radio (GCC : ${CMAKE_CXX_COMPILER}). This build may or not work. We highly recommend using GCC version ${GCC_MIN_VERSION} or more recent.")
     ENDIF()
+    # Release flags: -O3 enables tree vectorization
+    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -O2")
```

Comment:
I'm not convinced we should do debug builds at -O2. But that's a different change.

Reply:
Definitely not.

Reply:
So, if we're all in agreement, I'll add a separate change to change this to `-Og`. But like I said, that's not what we have right now, one change at  a time, yadda yadda.